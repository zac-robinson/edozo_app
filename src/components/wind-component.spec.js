import React from 'react';
import { shallow } from 'enzyme';

import Wind from './wind-component';

describe('Wind component', () => {
  it('should render correctly when straight on wind', () => {
    const windProp = {
      speed: 50,
      deg: 0
    };
    const component = shallow(<Wind wind={windProp} />);

    expect(component).toMatchSnapshot();
  });

  it('should render correctly when wind is at angle', () => {
    const windProp = {
      speed: 50,
      deg: 254
    };
    const component = shallow(<Wind wind={windProp} />);

    expect(component).toMatchSnapshot();
  });
});
