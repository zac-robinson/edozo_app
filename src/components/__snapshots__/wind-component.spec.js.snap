// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wind component should render correctly when straight on wind 1`] = `
<styled.div>
  Wind speed: 
  50
   m/s 
  <br />
  Wind direction: 
  N
</styled.div>
`;

exports[`Wind component should render correctly when wind is at angle 1`] = `
<styled.div>
  Wind speed: 
  50
   m/s 
  <br />
  Wind direction: 
  W
</styled.div>
`;
